### ISSUE TITLE
load_fakejapanese_vlgothic should be removed after #883

### ISSUE THREAD

**Issue by mihaitodor** [Winetricks/winetricks#899]
load_fakejapanese_vlgothic substitutes Meiryo and Meiryo UI with VL Gothic, which I think we don't need any more, since #883 will make the Meiryo fonts available. If we leave it in there, running allfonts will load this verb automatically, which is probably not what the users want.

Font substitutes are documented, albeit sketchy, [here](https://technet.microsoft.com/en-us/library/cc939627.aspx).

LE: I think Wine [will ignore](https://github.com/wine-mirror/wine/blob/7183dc7360f1f7a514f9649fced4171175f16f06/dlls/gdi32/freetype.c#L2408) the substitute if Meiryo is installed, but not sure how to test this. WDYT?

**Comment by austin987** [comment]
I don't think removing it is the right action. Instead, it should tell the user it's deprecated, and that `meiryo` should be used instead. Then, it should call meiryo, so that the user gets what they wanted.

**Comment by mihaitodor** [comment]
OK, makes sense. I created #905.

**Comment by kakurasan** [comment]
VL Gothic is free software but Meiryo is non-free. Microsoft says "You may use the fonts that accompany the PowerPoint Viewer only to display and print content from a device running a Microsoft Windows operating system". So I would like not to use the verb `meiryo` on my Linux box and I still need current `fakejapanese_vlgothic`.

How about using "conflicts=" instead of removing `fakejapanese_vlgothic`? 

> I think Wine will ignore the substitute if Meiryo is installed

Confirmed. When Meiryo is installed, it is used.

> it should tell the user it's deprecated, and that meiryo should be used instead. Then, it should call meiryo, so that the user gets what they wanted.

Again, I need current `fakejapanese_vlgothic`. It should not call `meiryo`.

**Comment by mihaitodor** [comment]
@kakurasan Uh, that's annoying. My assumption was that the PowerPoint Viewer fonts can be installed without issue :( Would you have a reference to this specific clause? I would like to review it in detail. 

I am OK to use the `conflicts=` functionality and prevent `allfonts` from installing `fakejapanese_vlgothic`. @austin987 what do you think would be best to do here?

**Comment by austin987** [comment]
We have both corefonts and liberation. We can have both fake and proprietary.

**Comment by mihaitodor** [comment]
OK, so then I was thinking that maybe having allfonts load both fonts and font replacements is not ideal. For example, load_fakejapanese_ipamona stomps over load_fakejapanese, which is probably not what the user wants. How about adding an optional flag to allfonts which makes it skip font replacements? Something like `--skip-font-replacements`? Or maybe something more generic, like `--blacklist` which allows the user to specify which verbs to skip?

**Comment by mihaitodor** [comment]
Closing since this is no longer relevant. #908 has fixed the consistency issues.
